Sources/BraintreePayPal/BTPayPalVaultRequest.swift

```diff
@@ -9,24 +9,44 @@ import BraintreeCore
 
     // MARK: - Public Properties
 
+    /// Optional: User email to initiate a quicker authentication flow in cases where the user has a PayPal Account with the same email.
+    public var userAuthenticationEmail: String?
+
+    // MARK: - Internal Properties
+
     /// Optional: Used to determine if the customer will use the PayPal app switch flow.
     /// Defaults to `false`.
     /// - Note: This property is currently in beta and may change or be removed in future releases.
-    public var enablePayPalAppSwitch: Bool
+    var enablePayPalAppSwitch: Bool = false
```

Comment:
Should we consider defaulting this to `true`? Do we even need this property at all or can we assume if the universal link is passed a merchant is opting into the flow?

Reply:
hmm yea that's a good point ~ imo i feel it's safe to remove this property since we can assume whether or not to launch the app switch flow based on if `universalLink` is passed in

Reply:
I think we are going to leave this as is for now to make it apparent to merchants that they are opting into this flow with both the `bool` and a `universalLink`. With just the `universalLink` it may make merchants think we are using that on the existing web flow (which we are not)